### ISSUE TITLE
tox_unstrap / tox_remove_tcp_relay

### ISSUE THREAD

**Issue by askmatey** [irungentoo/toxcore#1599]
Reducing the amount of active DHT nodes could reduce the amount of active UDP/TCP connections, especially on mobile clients like phones, laptops or tablets. The logic should be implemented in a client, and not toxcore itself.

However, my current solution to reduce the amount of active UDP/TCP connections is to (1) disconnect from the network (`tox_kill`) and (2) to reconnect to the DHT network (`tox_new`) with the desired/reduced amount of DHT nodes (`tox_bootstrap` / `tox_add_tcp_relay` if `use_udp=false`).

I wonder if it would be possible, to **hot reduce** the amount of connected DHT nodes within toxcore itself (something like `tox_unstrap` / `tox_remove_tcp_relay`)?

**Comment by xavierle** [comment]
why did you closed the issue ?

you changed your mind about he solution ?

**Comment by iphydf** [comment]
An `unbootstrap` function would not really work, because the nodes you bootstrap to are a very small subset of nodes you actually interact with. Also, the nodes you bootstrap to may not be in the list for long, if other, better (closer to your friends) nodes can be found. Removing TCP relays might work, but probably isn't very useful, either.

TL;DR: I don't think this is a solution to bandwidth problems.
